src/cpu/cpu_batch_normalization_list.cpp

```diff
@@ -1,5 +1,6 @@
 /*******************************************************************************
 * Copyright 2019-2020 Intel Corporation
```

Comment:
```suggestion
* Copyright 2019-2021 Intel Corporation
```

Reply:
Thank you for the suggestion. I'll fix it.